pcdm-ext/works.rdf

```diff
@@ -0,0 +1,63 @@
+<?xml version="1.0"?>
+<?xml-stylesheet type="text/xsl" href="rdfs2html.xsl"?>
+<rdf:RDF
+    xmlns:dcterms="http://purl.org/dc/terms/"
+    xmlns:owl="http://www.w3.org/2002/07/owl#"
+    xmlns:rdf="http://www.w3.org/1999/02/22-rdf-syntax-ns#"
+    xmlns:rdfs="http://www.w3.org/2000/01/rdf-schema#">
+
+  <rdf:Description rdf:about="http://pcdm.org/works#">
+    <dcterms:title xml:lang="en">Portland Common Data Model: Works Extension</dcterms:title>
+    <dcterms:modified rdf:datatype="http://www.w3.org/2001/XMLSchema#date">2016-02-16</dcterms:modified>
+    <dcterms:publisher rdf:resource="http://www.duraspace.org/"/>
+    <rdfs:seeAlso rdf:resource="https://github.com/duraspace/pcdm/wiki"/>
+    <rdfs:seeAlso rdf:resource="http://iiif.io/api/presentation/2.0/"/>
+    <rdfs:comment xml:lang="en">Ontology for a PCDM extension that adds subclasses of pcdm:Object
+      to refine the relationships between a work and its constituent parts.  Also establishes
+      links to the IIIF Presentation API.</rdfs:comment>
+    <owl:versionInfo>2016/02/16</owl:versionInfo>
+  </rdf:Description>
+
+  <rdfs:Class rdf:about="http://pcdm.org/works#Work">
+    <rdfs:label xml:lang="en">Work</rdfs:label>
+    <rdfs:comment xml:lang="en">
+      A work or intellectual entity, such as a book, film, dissertation, etc.
+      Works have member FileSets representing their physical structure (e.g., pages in a book),
+      and related TopRanges representing their logical structure or structures (e.g., sections
+      and chapters in a book).
+    </rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://pcdm.org/models#Object"/>
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/works#"/>
+  </rdfs:Class>
+
+  <rdfs:Class rdf:about="http://pcdm.org/works#FileSet">
+    <rdfs:label xml:lang="en">FileSet</rdfs:label>
+    <rdfs:comment xml:lang="en">
+      A group of related Files, typically a single master File and its derivatives.
+    </rdfs:comment>
+    <rdfs:subClassOf rdf:resource="http://pcdm.org/models#Object"/>
+    <rdfs:isDefinedBy rdf:resource="http://pcdm.org/works#"/>
+  </rdfs:Class>
+
+  <rdfs:Class rdf:about="http://pcdm.org/works#TopRange">
+    <rdfs:label xml:lang="en">TopRange</rdfs:label>
+    <rdfs:comment xml:lang="en">
+      A logical ordering of the component parts of a Work, corresponding to a IIIF Range with
```

Comment:
:+1: to skos:closeMatch (I suggested that to @jpstroop as the best we could do too)
But :ok_hand: to the change ... not sure that it's a useful addition as there's no inferencing, and developers are unlikely to read the ontology comments, rather than the human formatted documentation.